packages/fxa-settings/src/pages/ResetPassword/CompleteResetPassword/index.tsx

```diff
@@ -55,67 +61,125 @@ const CompleteResetPassword = ({
-      ) : (
-        <WarningMessage
-          warningMessageFtlId="complete-reset-password-warning-message-2"
-          warningType="Remember:"
-        >
-          When you reset your password, you reset your account. You may lose
-          some of your personal information (including history, bookmarks, and
-          passwords). That’s because we encrypt your data with your password to
-          protect your privacy. You’ll still keep any subscriptions you may have
-          and Pocket data will not be affected.
-        </WarningMessage>
-      )}
-
-      {/* Hidden email field is to allow Fx password manager
-          to correctly save the updated password. Without it,
-          the password manager tries to save the old password
-          as the username. */}
+      {hasConfirmedRecoveryKey === false &&
+        recoveryKeyExists === false &&
+        hasSyncDevices && (
+          <div
+            className={`bg-orange-50 rounded-lg text-sm mt-6 text-left rtl:text-right p-3`}
+            data-testid="warning-message-container"
+          >
+            <div className="flex font-semibold">
+              <WarnIcon
+                role="img"
+                className="flex-initial mr-2 rtl:ml-2 mt-1"
+              />
+              <h1 className="flex-1">
+                <FtlMsg id="password-reset-data-may-not-be-recovered">
+                  Resetting your password may delete your encrypted browser
+                  data.
+                </FtlMsg>
+              </h1>
+            </div>
+            <div className="ps-4 pb-6">
+              <p className="font-semibold pt-4">
+                <IconSyncDevice role="img" className="inline-block mr-2" />
+                <FtlMsg id="password-reset-previously-signed-in-device">
+                  Have a device where you previously signed in?
+                </FtlMsg>
+              </p>
+              <p className="ps-6 text-xs">
+                <FtlMsg id="password-reset-data-maybe-saved-locally">
+                  Your browser data may be locally saved on that device. Sign in
+                  there with your new password to restore and sync.
+                </FtlMsg>
+              </p>
+              <p className="font-semibold pt-4">
+                <IconNonSyncDevice role="img" className="inline-block mr-2" />
+                <FtlMsg id="password-reset-no-old-device">
+                  Have a new device but don’t have your old one?
+                </FtlMsg>
+              </p>
+              <p className="ps-6 text-xs">
+                <FtlMsg id="password-reset-encrypted-data-cannot-be-recovered">
+                  We’re sorry, but your encrypted browser data on Firefox
+                  servers can’t be recovered.
+                </FtlMsg>
+              </p>
+              <p className="ps-6 text-xs mt-4">
+                <a
+                  href="https://support.mozilla.org/en-US/kb/reset-your-firefox-account-password-recovery-keys"
```

Comment:
Wondering if this link should go to this link or this one: https://support.mozilla.org/en-US/kb/how-reset-your-password-without-account-recovery-keys-access-data

This second link would apply more to bullet 1 though.

Reply:
Seems better!